### ISSUE TITLE
Percentage listened or hours left?

### ISSUE THREAD

**Issue by jammekod** [GrakovNe/lissen-android#97]
I love the minimalist design and use the app daily and it is very esthetic, I was wondering though if it was possible to design in percentage or hours left? I know it is asking too much since the design is minimal and cool. Thanks in advance!

**Comment by GrakovNe** [comment]
Yes, it may be implemented. The app previously displayed a label for each book showing the duration, but it was removed during the redesign of the downloading feature, as it could not be supported for podcasts.

If I (or someone else) find a way to display the duration or remaining percentage without causing UI/UX issues, it will be implemented.

I'll try to handle this after the January holidays.

**Comment by jammekod** [comment]
Take your time and no hurry, I'm just sharing ideas 💡 I'll also try to translate the app into other languages in the upcoming weeks

**Comment by jammekod** [comment]
I want to minimize the time for you to reinvent the wheel, here is a screenshot I saw from an iOS app. Hope it helps! ;) ![68747470733a2f2f692e696d6775722e636f6d2f413149596135492e706e67.png](https://github.com/user-attachments/assets/ad0529ba-ac2b-4a7e-885c-53f879b60e53)

**Comment by GrakovNe** [comment]
Looks better that my implementation. Thank you much

**Comment by GrakovNe** [comment]
Done as you suggested in https://github.com/GrakovNe/lissen-android/releases/tag/1.2.3

Thanks a lot for the request!

Screenshot btw: 

![Image](https://github.com/user-attachments/assets/ada136ac-50cd-4556-ae37-b11437a2b5bf)
